### PR TITLE
IACO-679: Follow on feedback - sample backend

### PR DESCRIPTION
**Pull request by grozen**
This pull requests contains changes in light of the feedback gathered during the iOS party.

Namely, you can now use a rake task to set your own credentials without having to manually edit the code.

### CODE REVIEW

**Comment by rauchy** [comment]
Sorry to be a partypooper, but wouldn't a `API_KEY=foo API_SECRET=bar rackup` be a sufficient and simpler solution?

File: README.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ The server itself is implemented in [Ruby](https://www.ruby-lang.org/en/) using
 The server logs all API requests it performs to the terminal, so you can see what's going on even without diving into the code.
 
 ## Integration with the Klarna on Demand API
```
Comment (joaomilho) [inline README.md:9]: I guess "on demand" is kind of a single word. So "on Demand" seems odd to me. Ben & Case probably now better.

**Comment by tomerlevi444** [comment]
Approved.

File: README.md (hunk 1 above)
Comment (grozen) [inline README.md:9]: @joaomilho this is how we refer to the product in all our docs. We've actually run this past Ben in the past.

**Comment by yuval-netanel** [comment]
Approved.

**Pull request merged**